src/styles/main.scss

```diff
@@ -1,3 +1,65 @@
+$calendar-width: 707px;
+
+* {
+  box-sizing: border-box;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
 body {
   margin: 0;
 }
+
+.calendar {
+  width: $calendar-width;
+  margin: 100px auto;
```

Comment:
The calendar should be centered both horizontally and vertically. Maybe try to add display flex to parent, which is body